torchaudio/functional/functional.py

```diff
@@ -1215,7 +1206,7 @@ def compute_kaldi_pitch(
         recompute_frame: int = 500,
         snip_edges: bool = True,
 ) -> torch.Tensor:
-    """Extract pitch based on method described in [1].
```

Comment:
@mthrok and @carolineechen -- I do not see a discussion for the use of bibtex. Can you point to me where this is used in pytorch/text/vision? Was this discussed internally somewhere? Though I love bibtex for long free form document, this is not the appropriate choice here. On the web, this renders okay. However,
- python and ipython only show the :footcite: reference and the user wouldn't know where to look for the actual reference.
- this is inconsistent reference styling with other weblinks as they are still simply linked in the docstring, e.g. functional vad.

```
> help(torchaudio.functional.compute_kaldi_pitch)

Help on function compute_kaldi_pitch in module torchaudio.functional.functional:

compute_kaldi_pitch(waveform: torch.Tensor, sample_rate: float, frame_length: float = 25.0, frame_shift: float = 10.0, min_f0: float = 50, max_f0: float = 400, soft_min_f0
: float = 10.0, penalty_factor: float = 0.1, lowpass_cutoff: float = 1000, resample_frequency: float = 4000, delta_pitch: float = 0.005, nccf_ballast: float = 7000, lowpas
s_filter_width: int = 1, upsample_filter_width: int = 5, max_frames_latency: int = 0, frames_per_chunk: int = 0, simulate_first_pass_online: bool = False, recompute_frame:
 int = 500, snip_edges: bool = True) -> torch.Tensor
    Extract pitch based on method described in :footcite:`6854049`.
```

Reply:
#1575